stage1/init/init.go

```diff
@@ -225,11 +225,20 @@ func installAssets() error {
 	if err != nil {
 		return err
 	}
+	systemdJournaldBin, err := lookupPath("systemd-journald", "/usr/lib/systemd:/lib/systemd")
+	if err != nil {
+		return err
+	}
 
 	assets := []string{}
```

Comment:
Can you just initialize the array with all these values, all the appends seems silly